Libraries/CustomComponents/Navigator/NavigatorSceneConfigs.js

```diff
@@ -507,7 +523,10 @@ var BaseConfig = {
 var NavigatorSceneConfigs = {
   PushFromRight: {
     ...BaseConfig,
-    // We will want to customize this soon
+    animationInterpolators: {
+      into: buildStyleInterpolator(FromTheRight),
+      out: buildStyleInterpolator(ToTheLeftIOS),
```

Comment:
the default PushFromRight config includes `ToTheLeftIOS`, even on Android? Can we make this platform specific, or rename `ToTheLeftIOS`?